src/ensembl/webpack/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  root: true, // do not try to use eslint rules from parent folder
```

Comment:
Because the parent eslint config expects typescript, and has certain rules for typescript (e.g. I remember it being unhappy about using the `require` statement).